metadata/climate/nws_forecast_zones.md

```diff
@@ -0,0 +1,74 @@
+# Title
+
+Utah NWS Forecast Zones
+
+## ID
+
+92b56cc3-e822-4d6f-ae3f-1c750498a5bf
+
+## Brief Summary
+
+Polygon dataset of zones used to estimate weather conditions for various regions throughout Utah.
+
+## Summary
+
+This dataset includes polygon boundaries for the National Weather Service (NWS) Forecast Zones found within Utah. This dataset includes forecast zones only and does include other climate information such as surface temperature or precipitation.
+
+## Description
+
+### What is the dataset?
+
+These data represent the [National Weather Service](https://www.weather.gov/) (NWS) Forecast Zone boundaries within the state of Utah. The NWS issues forecasts and some watches and warnings for these zones, which often align with county boundaries, but in many cases are subsets of counties. Counties are broken or combined into zones to account for differences in weather trends due to topographic factors.
+
+### What is the purpose of the dataset?
+
+This dataset has been made available to the public for general analytic and cartographic purposes.
+
+<!--- Do we know if any Utah-specific companies or entities use these zones for their weather reporting? Do these data have any other specific purposes? --->
+
+### What does the dataset represent?
+
+Each polygon feature in this dataset represents the approximate geographic boundaries of a NWS zone. Features in this dataset include attributes such as the time_zone, name, coordinates, as well as several NWS codes.
+
+### How was the dataset created?
+
+UGRC created this dataset from a [nationwide](https://www.weather.gov/gis/PublicZones) dataset provided by the National Weather Service. We projected the data into UTM Zone 12N ([EPSG: 26912](https://epsg.io/26912)) and clipped the forecast zones to the Utah state boundary. These zones are updated as new information becomes available from the NWS.
+
+<!--- The original metadata specified the EPSG code for the coordinate system, which is something I haven't seen repeated in our other datasets. Is that an essential detail to include for this dataset in particular? --->
```

Comment:
EPSG is the organization that creates spatial references more or less https://en.wikipedia.org/wiki/EPSG_Geodetic_Parameter_Dataset. They are synonymous with WKID and SRID. 26912 is the state standard that we store the data. All of the facets of the SGID do not use 26912. ArcGIS Online for instance uses Web Mercator which is 3857.